src/audit-license-headers.js

```diff
@@ -29,25 +29,24 @@ var repoutil = require('./repoutil');
 
 // constants
 var COMMON_RAT_EXCLUDES = [
+    // Binrary Files
     '*.wav',
     '*.webloc',
-    '*jasmine-1.2.0*',
-    'jasmine-2.0.0',
-    'topcoat-0.7.5',
+
+    // iOS Related
+    '*.xcworkspacedata',
+    '*.xccheckout',
+    '*.xcscheme',
     '*.xcodeproj',
-    '.*',
     '*-Info.plist',
     'Info.plist',
+
+    // Other
+    '.*',
+    '*.json', // Excludes all JSON files because commenting is not supported.
```

Comment:
```suggestion
    '*.json', // Excludes all JSON files because commenting is not supported.
    '*.txt', // Excludes all text files because commenting is not supported.
```